### PR TITLE
CHANGELOG: Add 2.0.8 change logs

### PR DESCRIPTION
**Pull request by winkyao**
Add release note for v2.0.8

@shenli @lilin90 @zz-jason PTAL

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV changelog](https://github.com/pingcap/tikv/blob/master/CHANGELOG.md) and [PD changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.8] - 2018-10-16
+### Improvements
+- Slow down the AUTO-ID increasing speed when the `Update` statement does not modify the corresponding AUTO-INCREMENT column [#7846](https://github.com/pingcap/tidb/pull/7846)
```
Comment (shenli) [inline CHANGELOG.md]: s/AUTO-ID/AUTO_INCREMENT ID/
s/AUTO-INCREMENT/AUTO_INCREMENT/

**Comment by winkyao** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM

**Comment by winkyao** [comment]
@zz-jason PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+### Bug fixes
+- Quickly create a new etcd session to recover the service when the PD leader goes down [#7810](https://github.com/pingcap/tidb/pull/7810)
+- Fix the issue that the time zone is not considered when the default value of the `DateTime` type is calculated [#7672](https://github.com/pingcap/tidb/pull/7672)
+- Fix the issue that `duplicate key update` inserts values incorrectly in some conditions [#7685](https://github.com/pingcap/tidb/pull/7685)
+- Fix the issue that the predicate conditions of `UnionScan` are not pushed down [#7726](https://github.com/pingcap/tidb/pull/7726)
+- Fix the issue that the time zone is not correctly handled when you add the `TIMESTAMP` index [#7812](https://github.com/pingcap/tidb/pull/7812)
+- Fix the memory leak issue caused by the statistics module in some conditions [#7864](https://github.com/pingcap/tidb/pull/7864) 
+- Fix the issue that the results of `ANALYZE` cannot be obtained in some abnormal conditions [#7871](https://github.com/pingcap/tidb/pull/7871) 
+- Do not unfold expressions in `SYSDATE`, to ensure the returned results are correct [#7894](https://github.com/pingcap/tidb/pull/7894)
```
Comment (zz-jason) [inline CHANGELOG.md]: s/unfold/fold/

**Comment by zz-jason** [comment]
/run-all-tests tidb-test=release-2.0 tikv=release-2.0 pd=release-2.0

**Comment by iamxy** [comment]
/run-sqllogic-test tidb-test=release-2.0 tikv=release-2.0 pd=release-2.0

**Pull request merged**